cmake/modules/packages/FindKDU.cmake

```diff
@@ -39,6 +39,24 @@ if(KDU_ROOT AND EXISTS "${KDU_ROOT}/coresys")
         find_library(KDU_AUX_LIBRARY ${KDU_AUX_SHARED_LIB_NAME}
                      PATH ${KDU_ROOT}/lib/Linux-x86-64-gcc)
 
+    elseif("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "aarch64" AND "${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+        file(READ "${KDU_ROOT}/coresys/make/Makefile-Linux-arm-64-gcc" CORESYS_MAKEFILE_CONTENTS)
```

Comment:
as this logic is mostly common with the x86_64 case but the Linux-x86-64-gcc vs Linux-arm-64-gcc difference, couldn't that be factored into a function / macro ?